tests/test-rest-attachments-controller.php

```diff
@@ -112,6 +112,7 @@ public function test_registered_query_params() {
 			'context',
 			'exclude',
 			'filter',
+			'filter[posts_per_page]',
```

Comment:
Okay, now I see what you did.  I don't know if it should be done that way. See my above comment for line 1705 in class-rest-posts-controller.php